### ISSUE TITLE
Typo in ARIA attribute

### ISSUE THREAD

**Issue by digitalbricks** [teppokoivula/SearchEngine#28]
In [line 197 of SearchEngine.module.php](https://github.com/teppokoivula/SearchEngine/blob/f757306c7f0ad6aad4d44f136e4e3262ee891730/SearchEngine.module.php#L197) is a typo in the aria attribute:

_aria-labelled-by_ should be _aria-labelledby_

(This small typo was found in a A11y audit peformed on a customer website.)

**Comment by digitalbricks** [comment]
For others stumbling over this:
As a temporary solution you can overwrite the output using the code before loading the module:

```php
$config->SearchEngine = [
    'render_args' => [
        'templates' => [
            'results_list' => '<ul class="{classes.results_list}" aria-labelledby="{results_summary_id}">%s</ul>',
        ]
    ]
];
```

**Comment by teppokoivula** [comment]
Thanks @digitalbricks, this is fixed now in 0.38.1.
